moonstone/filters/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .filtering import (  # noqa
```

Comment:
Maybe we could rename `filters` module to `filtering` and reorganize a bit the directory.

For instance:

```bash
/filtering
  __init__.py
  base.py
  mean_filtering.py
  ... (other filtering modules extracted from the existing `Filtering` module
```